File: domain/src/main/java/io/prizy/domain/contest/ports/ContestSubscriptionPublisher.java
package io.prizy.domain.contest.ports;

import io.prizy.domain.contest.event.ContestSubscriptionCreated;

/**
 *  @author Nidhal Dogga
 *  @created 4/24/2022 12:53 AM
 */

public interface ContestSubscriptionPublisher {
  void publish(ContestSubscriptionCreated event);
}

File: domain/src/main/kotlin/io/prizy/domain/contest/ports/ContestSubscriptionPublisher.kt
package io.prizy.domain.contest.ports

import io.prizy.domain.contest.event.ContestSubscriptionCreated

/**
 *  @author Nidhal Dogga
 *  @created 4/24/2022 12:53 AM
 */

interface ContestSubscriptionPublisher {
  suspend fun publish(event: ContestSubscriptionCreated)
}
